<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>642</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>642</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>VLR patch automatique</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ressource.qrc">
    <normaloff>:/ressource/DreamteamLogo.ico</normaloff>:/ressource/DreamteamLogo.ico</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_process">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777211</height>
     </size>
    </property>
    <property name="text">
     <string>Démarrer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_browse">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_done">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>40</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Terminé !</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_process">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>40</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>in process...</string>
    </property>
   </widget>
   <widget class="FileEdit" name="fileEdit_path">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>541</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_choix_fichier">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Choisir fichiers</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_choix_image">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Choisir images</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_fichiers">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>441</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Update tous les fichiers textes</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_imagesDDS">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>441</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Update toutes les images du dossier DDS</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_imagesZIP">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>521</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Update tous  les DDS (images de fond)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_videos">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>501</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Update les vidéos</string>
    </property>
   </widget>
   <zorder>pushButton_process</zorder>
   <zorder>pushButton_browse</zorder>
   <zorder>fileEdit_path</zorder>
   <zorder>pushButton_choix_fichier</zorder>
   <zorder>pushButton_choix_image</zorder>
   <zorder>label_done</zorder>
   <zorder>label_process</zorder>
   <zorder>checkBox_fichiers</zorder>
   <zorder>checkBox_imagesDDS</zorder>
   <zorder>checkBox_imagesZIP</zorder>
   <zorder>checkBox_videos</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileEdit</class>
   <extends>QLineEdit</extends>
   <header>fileEdit</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pushButton_browse</tabstop>
  <tabstop>pushButton_process</tabstop>
 </tabstops>
 <resources>
  <include location="ressource.qrc"/>
 </resources>
 <connections/>
</ui>
